### DISCUSSION TITLE
Requesting the possession of the `news` crate name.

### DISCUSSION THREAD

**Discussion by samyosm** [rust-lang/crates.io#6739] [General]
I'm currently making a application that will allow people to consume the news via a text user interface and I wanted it to be called `news`. Therefore, seeing that the [news crate](https://crates.io/crates/news) is available to the first who requests it to the crates.io team, I would like to request possession of it. 

## UI
<img width="1005" alt="shapes" src="https://github.com/rust-lang/crates.io/assets/99157490/cf580730-442e-48c7-a06d-5dec247da0fb">

**Comment by sohaibaumer** [comment]
To request ownership of the "news" crate on crates.io, contact the crates.io team and explain your project's purpose. Express your interest in using the name "news" for your text user interface application. Await their guidance and approval.

**Reply by samyosm** [reply]
How do I contact them? The news crate told me to come here to ask for it.

**Reply by LawnGnome** [reply]
@samyosm Please send an e-mail to help@crates.io.

**Reply by samyosm** [reply]
Thank you!
